### ISSUE TITLE
Javascript i18n calls need to be updated

### ISSUE THREAD

**Issue by robbieaverill** [silverstripe/silverstripe-comments#226]
When viewing a page with the comments extension applied in the CMS, you see the following errors in the console:

```
GET http://ss4.localhost/resources/silverstripe/comments/thirdparty/jquery-validate/jquery.validate.min.js?m=1513219228 net::ERR_ABORTED


Class ss.i18n not defined in en.js?m=1513574621:4
(anonymous) @ en.js?m=1513574621:4
```

We need to ensure that [the i18n shiv](https://github.com/silverstripe/silverstripe-admin/blob/1.0/client/dist/js/i18n.js) is included with Requirements from the admin module before adding i18n javascript.

The thirdparty lib may need to be "exposed" via composer as well

**Comment by NightJar** [comment]
This seems to be a non-thing now as `silverstripe/admin : client/dist/js/bundle.js` packages `i18n` within it.
The front end presentation however still had the issue, and is fixed in #227

**Comment by robbieaverill** [comment]
Sorry I should’ve been clearer. This is a frontend only issue
